huntflow_api_client/models/request/resume.py

```diff
@@ -6,17 +6,17 @@
 
 
 class ApplicantResumeUpdateData(BaseModel):
-    body: Optional[str] = Field(..., description="Resume text")
+    body: Optional[str] = Field(None, description="Resume text")
 
 
 class ApplicantResumeUpdateRequest(JsonRequestModel):
-    account_source: Optional[PositiveInt] = Field(..., description="Resume source ID")
+    account_source: Optional[PositiveInt] = Field(None, description="Resume source ID")
     data: Optional[ApplicantResumeUpdateData] = Field(
-        ...,
```

Comment:
Do you intentionally change the field's signature? `...` -> `None`

Reply:
No 😢  Looks like bump-pydantic tricks.